src/install_host_app.sh

```diff
@@ -126,16 +140,26 @@ else
   curl -sSL "$MANIFEST_URL" > "$MANIFEST_FILE_PATH"
 fi
 
-# When using sed on macOS, backup extension is an mandatory argument
-#   whereas on GNU sed or BSD sed backup extension may be omit.
-if [ "$KERNEL_NAME" == 'Darwin' ]; then
+if [ "$IS_BSD" = true ]; then
+  # Use BSD style sed on BSD-ish systems
```

Comment:
We learned from https://github.com/passff/passff-host/pull/12, that  BSD and Darwin shouldn't be handled together. This is probably not going to work.

Reply:
I was actually surprised to see `-i` working at all for BSDs, but apparently OS X and OpenBSD now support in place replacement. I don't know what the status is for other BSDs like Free or Net.

To be honest, though, I was half considering replacing that `if` with a plain POSIX `sed` use that would work in any situation. Should be easy enough to pull of, and that'll clear compatibility concerns.